Add meeting leaders to team listings + refactors (#92)
QuiltMC/quiltmc.org: src/pages/en/about/teams.mdx

<TeamMembers
	type="moderator"
	list={[
		"Ember",
		"Forkk",
		"Larry",
		"Octal",
		"SilverAndro",
		"Southpaw",
		"snoozestudios",
		"tempest",
		"Redblueflame",
	]}
/>

## Events Team

The Events Team is responsible for organising and managing events within our community spaces, which can include anything from AMAs to fully-blown online modding conventions. While they aren't Moderators, they are always backed up by the rest of the Community Team, and also have the ability to directly moderate events.

<TeamMembers
	type="community"
	list={["Fusion", "Lemma", "Octal", "Bubblie", "Pyrrha"]}
/>

### Meeting Leaders

Meeting Leaders are in charge of our monthly developer meetings.
Their responsibilities include organizing, leading and guiding meetings, collecting talking points from developers and making any changes to our meetings system.

> <TeamMembers type="community" list={["Bubblie"]} />

Reviewer: Gonna note that @TibiNonEst is also a meeting leader, though I presume their addition to the listing is pending their graduation from trainee status.

Author: > though I presume their addition to the listing is pending their graduation from trainee status.
Yeah exactly

Reviewer: Noting that they've graduated from their trainee status as of today, so they should be added to the Meeting Leader listing now.

## Outreach Team

> The outreach team is responsible for communications outside of and within Quilt. They write blog posts, manage social media and website content, among other things.

Reviewer: The second sentence seems a bit awkward now, due to listing only two things. In lieu of adding a third thing (because I don't know of a third thing to add; that would need answering by someone from the team), the two sentences could be merged together instead.
```suggestion
The outreach team is responsible for communications outside of and within Quilt, including writing blog posts and managing social media and website content.
```

Author: They're also responsible for our podcasts :p 
Will add that to the list

This team is responsible for the following projects:

<ProjectButtons names={["asmr-processor-prototype", "chasm"]} />

<TeamMembers
	type="developer"

> 	list={["CheaterCodes", "Earthcomputer", "Kroppeb", "Pyrofab"]}

Reviewer: This seems to be missing @FoundationGames, based on the `Team: Chasm` role in the Toolchain discord.

Reviewer 2: I asked Foundation for card information back in September, they still haven't responded.
